lib/splitclient-rb/sse/workers/splits_worker.rb

```diff
@@ -32,25 +32,44 @@ def stop
           SplitIoClient::Helpers::ThreadHelper.stop(:split_update_worker, @config)
         end
 
-        def add_to_queue(change_number)
-          @config.logger.debug("feature_flags_worker add to queue #{change_number}")
-          @queue.push(change_number)
+        def split_update(notification)
```

Comment:
you should check `return if @splits_repository.get_change_number.to_i > notification.data['changeNumber']`

Reply:
will do